activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -915,7 +914,7 @@ def configure_connection
         variables['wait_timeout'] = self.class.type_cast_config_to_integer(wait_timeout)
 
         # Make MySQL reject illegal values rather than truncating or blanking them, see
-        # http://dev.mysql.com/doc/refman/5.6/en/server-sql-mode.html#sqlmode_strict_all_tables
+        # http://dev.mysql.com/doc/refman/5.6/en/sql-mode.html#sqlmode_strict_all_tables
```

Comment:
This is an unrelated change to removing the dev.rubyonrails.org links, can you please do it on a separate commit instead?